package.json

```diff
@@ -45,6 +47,7 @@
     "@types/jquery": "^3.5.13",
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
+    "auto-changelog": "^2.4.0",
```

Comment:
`auto-changelog` should be a dev-dependency for it won't be part of a final build.

So add the dependency by `yarn add auto-changelog --dev` instead.

Reply:
The `auto-changelog` is in the `devDependencies`.

Reply:
Sorry, my mistake 😆